packages/react-devtools-scheduling-profiler/src/view-base/VerticalScrollOverflowView.js

```diff
@@ -1,3 +0,0 @@
-// TODO Vertically stack views (via verticallyStackedLayout).
```

Comment:
was this file intentionally deleted?

Reply:
It was a placeholder view that ended up being moved to `packages/react-devtools-scheduling-profiler/src/view-base/vertical-scroll-overflow/VerticalScrollOverflowView.js`